imap/mboxname.c

```diff
@@ -577,8 +577,33 @@ EXPORTED mbname_t *mbname_from_intname(const char *intname)
     return mbname;
 }
 
-EXPORTED mbname_t *mbname_from_extname(const char *extname, const struct namespace *ns, const char *userid)
+EXPORTED mbname_t *mbname_from_extname(const char *extname,
+                                       const struct namespace *ns,
+                                       const char *userid)
 {
+    char *freeme = NULL;
```

Comment:
There's some early returns in the unmodified portion of this function, which I think may result in `freeme` being leaked.  Github won't let me comment on them directly though, because the patch doesn't change them... they start around `/* specialuse magic */`

Reply:
Yeah, thanks for catching that.  Should be fixed now.